vmm/src/api/http/http_endpoint.rs

```diff
@@ -163,6 +163,38 @@ macro_rules! vm_action_put_handler_body {
     };
 }
 
+macro_rules! vm_action_put_handler_body_with_fds {
```

Comment:
Looks like this refactoring is not used and can be dropped.

Reply:
I thought this is good to have. But, Sure, will remove it.